Utilities/constants.py

```diff
@@ -0,0 +1,34 @@
+"""
+All of the constant values used across all of the scripts are located here.
+"""
+
+# Safety
+DEFAULT_VELOCITY = 0.50
+VELOCITY_THRESHOLD = 2  # never let the drone go faster than 1 m/s for safety (is this a good number?)
+MINIMUM_ALLOWED_ALTITUDE = 0.5
+MAXIMUM_ALLOWED_ALTITUDE = 4.0
+RANGEFINDER_MIN = 0.29
+RANGEFINDER_EPSILON = 0.03
+
+# Connection-oriented
+CONNECTION_STRING_SIMULATOR = "tcp:127.0.0.1:5762"
+CONNECTION_STRING_REAL = "/dev/serial/by-id/usb-3D_Robotics_PX4_FMU_v2.x_0-if00"
```

Comment:
You should cite a source for this constant

Reply:
I am not sure on the source for either of them. I copied the simulator one from the original flight code that existed before I started working on flight and the second one from Innocent. I tried looking online but can't find any source.

Reply:
Ask Innocent, he may know.

Reply:
I was able to get sources from Innocent and I have commented them in the code.